daphne/src/error/aborts.rs

```diff
@@ -106,6 +106,27 @@ pub enum DapAbort {
 }
 
 impl DapAbort {
+    pub fn as_kind_str(&self) -> &'static str {
+        match self {
+            DapAbort::BadRequest(_) => "bad request",
+            DapAbort::BatchInvalid { .. } => "batchInvalid",
+            DapAbort::BatchMismatch { .. } => "batchMismatch",
+            DapAbort::BatchOverlap { .. } => "batchOverlap",
+            DapAbort::Internal(_) => "internal error",
```

Comment:
is it possible to camelCase these 2 (`BadRequest` and `Internal`) for consistency?

Reply:
I didn't do it so that it retained the old behavior as much as possible, but it did bother me as well :sweat_smile: